roles/nf-core/tasks/pipeline.yml

```diff
@@ -49,10 +48,11 @@
   lineinfile:
     dest: "{{ ngi_pipeline_conf }}/{{ bash_env_script }}"
     line: >
-          alias {{ pipeline }}='nextflow run {{ sw_path }}/{{ pipeline }}/workflow/ \
-          -profile uppmax \
-          -c {{ ngi_pipeline_conf }}/nextflow_miarka_{{ site }}.config \
-          -c {{ ngi_pipeline_conf }}/{{ pipeline }}_{{ site }}.config'
+      alias {{ pipeline }}='nextflow run {{ sw_path }}/{{ pipeline }}/workflow/ \
+      --custom_config_base {{ sw_path }}/{{ pipeline }}/configs/ \
+      -profile uppmax \
+      -c {{ ngi_pipeline_conf }}/nextflow_miarka_{{ site }}.config \
+      -c {{ ngi_pipeline_conf }}/{{ pipeline }}_{{ site }}.config'
```

Comment:
do you know if this will actually work in practice? the reason I'm a bit hesitant is that I believe `--custom_config_base` is a pipeline parameter whereas `-c` is a Nextflow parameter and I wouldn't be surprised if this has an effect on precedence.. can be tested easily enough of course...